README.md

```diff
@@ -48,11 +48,21 @@ Ensure docker is running. Spin up MySQL locally:
 ```
 $ ./docker/run-mysql.sh start leonardo  
 ```
-Build Leonardo and run tests.
+Build Leonardo and run all unit tests.
 ```
 export SBT_OPTS="-Xmx2G -Xms1G -Dmysql.host=localhost -Dmysql.port=3311"
 sbt clean compile "project http" test
 ```
+You can also run a particular test suite, e.g.
+```
+sbt "project http" "testOnly *LeoAuthProviderHelperSpec"
```

Comment:
`project http` is not necessary needed. Doesn't hurt to have it either

Reply:
Ah...it's interesting that I get 
```
[error] (automation / Test / test) sbt.TestsFailedException: Tests unsuccessful
```
with 
```
sbt clean compile test
```
i.e. with `"project http"` omitted, even though all tests pass. But I don't get that with the `testOnly` command line.